test/runtests.jl

```diff
@@ -258,6 +258,14 @@ const PyInt = pyversion < v"3" ? Int : Clonglong
     end
     @test convert(BigInt, PyObject(1234)) == 1234
 
+    # hasproperty, getproperty, and propertynames
+    @test PyCall.hasproperty(np, "random")
```

Comment:
These tests are failing on Windows because numpy is not installed by default:
https://ci.appveyor.com/project/StevenGJohnson/pycall-jl-nu3aa/build/1.0.342/job/3t0m1me5bkffvn3t#L113

If you need to test it against Python modules, you should use something from Python standard library.  If you need modules with three levels to test access pattern like `A.B.C` (and that is Python 2/3 compatible), use, e.g., `logging.config.dictConfig`:
https://docs.python.org/2/library/logging.config.html

But if you just need to have nested Python object, it's probably the easiest to create nested class:

```julia
py"""
class A:
    class B:
        C = 1
"""
A = py"A"
@test A.B.C == 1
```